src/gfn/utils/distributions.py

```diff
@@ -49,51 +47,16 @@ def log_prob(self, sample: torch.Tensor) -> torch.Tensor:
 class GraphActionDistribution(Distribution):
     """A mixture distribution."""
 
-    def __init__(self, logits: TensorDict):
+    def __init__(self, probs: TensorDict):
         """Initializes the mixture distribution.
 
         Args:
-            logits: A TensorDict of logits.
+            probs: A TensorDict of probs.
```

Comment:
Where is the normalization coming from?

Reply:
Oh, I've seen you've moved the logic elsewhere. I'm curious why you would prefer the transformation of logits to probs there, as opposed to here (to me, this seems the logical spot, but I'm open to suggestions).